.gitignore

```diff
@@ -4,8 +4,13 @@ es*/compilers/*
 .ruby-version
 _site/
 .idea/
+.history/
+.project
```

Comment:
i'm not sure what these are, but specific tools' files should be in your global gitconfig, not added to every project you touch

Reply:
.project is from Eclipse and it's required to have any project in your workspace. In this case it can be omitted, as there's no relevant info in it, but that is not always the case, so it cannot be excluded globally, I guess similarly to `.idea/`?

.history is a VS Code (plugin?) thing which I think makes sense to exclude globally, will check

Reply:
`.idea/` is just a default from github, it shouldn't be there either.

why can't `.project` be excluded in your global `.gitignore`?

Reply:
See https://einverne.github.io/post/2016/04/should-i-git-ignore-project-file-under-eclipse.html

Basically: for many projects in Eclipse you want the .project file to be committed, hence it shouldn't be excluded globally

Should I remove `.idea/` then?

Reply:
That seems like an unfortunate design in Eclipse :-/

sure, let's remove `.idea/`

Reply:
done

Reply:
`.project` still should be removed, though, despite the inconvenience for yourself.

Reply:
Done